src/File/UploadFromFile.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * @copyright 2015-2017 Contentful GmbH
+ * @license   MIT
+ */
+
+namespace Contentful\File;
+
+class UploadFromFile implements FileInterface
```

Comment:
Would it make sense to add another interface `UploadableFileInterface` for this class and `UploadFile` so we can type hint for it in the CMA?